[E2E/Acceptance CI Failure]: Timeout error in Acceptance (logged-in-user/view-subtopic-study-guides)

### CI Test Type

Acceptance

### Which CI step failed?

Acceptance (logged-in-user/view-subtopic-study-guides)

### Did the test fail after an automatic rerun?

Test failed once but was not automatically re-run

### Did the test fail when running on desktop, mobile, or both?

Desktop

### Stacktrace

```shell
INFO     2025-12-13 15:53:47,671 module.py:830] default: "GET /learnerdashboardexplorationsprogresshandler/data HTTP/1.1" 200 354
INFO     2025-12-13 15:53:47,697 module.py:830] default: "GET /learnerdashboardtopicsandstoriesprogresshandler/data HTTP/1.1" 200 2875
INFO     2025-12-13 15:53:47,765 module.py:830] default: "GET /subtopic_mastery_handler/data?selected_topic_ids=%5B%5D HTTP/1.1" 200 133
INFO     2025-12-13 15:53:47,778 module.py:830] default: "GET /build/assets/images/background/bannerD.0c10c46ee1bc27187acb776daabf3f7f.svg HTTP/1.1" 200 52069
INFO     2025-12-13 15:53:47,809 module.py:830] default: "GET /assetsdevhandler/topic/5svtZsKvZnkD/assets/thumbnail/img_20251213_154726_vhxmq8kha1_height_150_width_128.svg HTTP/1.1" 200 74456
[test-log] 53:53.891: Screen recording stopped for curriculumAdm.
[test-log] 53:53.974: Browser closed for curriculumAdm.
[test-log] 54:16.038: Error while stopping screen recording for superAdm: Error: Screen recorder stop timed out
[test-log] 54:16.086: Browser closed for superAdm.
[test-log] 54:16.087: All browsers closed.
FAIL core/tests/puppeteer-acceptance-tests/specs/logged-in-user/view-subtopic-study-guides.spec.ts (537.365 s)
  Logged-in User
    ✕ should be able to view the updated study guides

  ● Logged-in User › should be able to view the updated study guides

    thrown: "Exceeded timeout of 500000 ms for a hook.
    Use jest.setTimeout(newTimeout) to increase the timeout value, if this is a long-running test."

      34 |   let releaseCoordinator: ReleaseCoordinator;
      35 |
    > 36 |   beforeAll(async function () {
         |   ^
      37 |     curriculumAdmin = await UserFactory.createNewUser(
      38 |       'curriculumAdm',
      39 |       'curriculumAdmin@example.com',

      at specs/logged-in-user/view-subtopic-study-guides.spec.ts:36:3
      at Object.<anonymous> (specs/logged-in-user/view-subtopic-study-guides.spec.ts:31:1)

Test Suites: 1 failed, 1 total
Tests:       1 failed, 1 total
Snapshots:   0 total
Time:        539.022 s, estimated 578 s
Ran all test suites matching /core\/tests\/puppeteer-acceptance-tests\/specs\/logged-in-user\/view-subtopic-study-guides.spec.ts/i.
INFO     2025-12-13 15:54:16,174 shutdown.py:48] Shutting down.
Error: 2-13 15:54:16 +0000] [9126] [ERROR] Worker (pid:9128) was sent SIGTERM!
Error: 2-13 15:54:16 +0000] [8972] [ERROR] Worker (pid:8974) was sent SIGTERM!
Error: 2-13 15:54:16 +0000] [8766] [ERROR] Worker (pid:8768) was sent SIGTERM!
Error: 2-13 15:54:16 +0000] [8736] [ERROR] Worker (pid:8740) was sent SIGTERM!
Error: 2-13 15:54:16 +0000] [8842] [ERROR] Worker (pid:8845) was sent SIGTERM!
ERROR    2025-12-13 15:54:16,318 module.py:958] Request to '/all_classrooms_summary' failed
INFO     2025-12-13 15:54:16,318 module.py:830] default: "GET /all_classrooms_summary HTTP/1.1" 500 -
INFO     2025-12-13 15:54:16,370 module.py:830] default: "GET /third_party/generated/webfonts/fa-solid-900.woff2 HTTP/1.1" 304 -
INFO     2025-12-13 15:54:16,503 module.py:830] default: "POST /frontend_errors HTTP/1.1" 500 161
INFO     2025-12-13 15:54:17,008 stub_util.py:321] Applying all pending transactions and saving the datastore
INFO     2025-12-13 15:54:17,008 stub_util.py:324] Saving search indexes
                                                                                    
 
i  emulators: Received SIGTERM for the first time. Starting a clean shutdown.
i  emulators: Please wait for a clean shutdown or send the SIGTERM signal again to stop right now.
i  Automatically exporting data using --export-on-exit "/home/runner/work/oppia/oppia/../firebase_emulator_cache" please wait for the export to finish...
⚠  Emulator UI has exited upon receiving signal: SIGTERM
i  Found running emulator hub for project dev-project-id at http://localhost:4400
i  Exporting data to: /home/runner/work/o
```

### Screenshots and/or Screen Recordings

https://github.com/user-attachments/assets/baf4d2e3-9217-4b69-80a7-167a7005bb3e
https://github.com/user-attachments/assets/9e272026-68d4-49c9-adf2-a1b95f2481f5

### Occurrences

https://github.com/oppia/oppia/actions/runs/20193489314?pr=24032

13 dec 2025

### Additional Information

_No response_

### Debugging document link

_No response_

--- a/core/tests/puppeteer-acceptance-tests/specs/logged-in-user/view-subtopic-study-guides.spec.ts
+++ b/core/tests/puppeteer-acceptance-tests/specs/logged-in-user/view-subtopic-study-guides.spec.ts
@@ -72,7 +72,7 @@ describe('Logged-in User', function () {
     );
 
     // Setup taking longer than 300000ms.
-  }, 500000);
+  }, 600000);
 
   it(
     'should be able to view the updated study guides',

